.github/workflows/test.yml

```diff
@@ -28,6 +28,26 @@ jobs:
       - name: Run lint
         run: pnpm lint
 
+  test_helpers:
```

Comment:
Need an extra step because we cannot run the tests for the files that are real ESM in all node versions.